apischeme_SSS.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+import subprocess
+import socket
+import sys
+import time
+import yaml
+import os
+
+
+from kubernetes import client, config
+from kubernetes.client import ApiClient, Configuration
+
+from openshift.dynamic import DynamicClient
+
+APISCHEME_SSS_NAME = "cloud-ingress-operator-apischeme"
+
+#####
+
+# from kubeconfig
+#k8s_client = config.new_client_from_config()
+#dyn_client = DynamicClient(k8s_client)
+
+# in cluster client creation
+k8s_client = ApiClient(config.load_incluster_config())
+dyn_client = DynamicClient(k8s_client)
+
+def get_hive_ips():
+    nodes = dyn_client.resources.get(api_version='v1', kind='Node')
+
+    hive_ips = []
+    for node in nodes.get().items:
+        for a in node.status.addresses:
+            if a.type == "ExternalIP":
+                # add /32 to the end since they're not recorded as CIDR blocks
+                hive_ips.append("{}/32".format(a.address))
+
+    if len(hive_ips) == 0:
+        print("not enough hive ips")
+        sys.exit(1)
+    return hive_ips
+
+def get_bastion_ips():
+    bastion_ip_str = os.getenv("ALLOWED_CIDR_BLOCKS",[])
+    bastion_ips = bastion_ip_str.split(",")
+    if len(bastion_ips) == 0:
+        print("not enough bastion ips")
+        sys.exit(1)
+    return bastion_ips
+
+apischeme_sss = """
+apiVersion: hive.openshift.io/v1
+kind: SelectorSyncSet
+metadata:
+ labels:
+   managed.openshift.io/osd: 'true'
```

Comment:
```suggestion
   managed.openshift.io/osd: "true"
```

match quote usage on line 61

Reply:
Data point: I believe `generate_template.py` will put single quotes in the generated output regardless of what's on the input side.

Reply:
changed, at least for consistency